package/CHANGELOG

```diff
@@ -82,6 +82,7 @@ Enhancements
    (Issue #4673) 
  * enables parallelization for analysis.dssp.dssp.DSSP (Issue #4674)
  * Enables parallelization for analysis.hydrogenbonds.hbond_analysis.HydrogenBondAnalysis (Issue #4664)
+ * Added pytest case to cover Raise errors cases frames and step/stop/step are present (Issue #4648)
```

Comment:
May I ask that we not add changelog entries for singular test additions. The changelog is really meant to be user facing "something has changed in the code". Large test changes are ok, but single test changes will clog up the changelog.

Reply:
Ah my bad, didn't know about that, but makes sense, removed the line now, should be fine now :)

Reply:
I agree with this.